hudi-client/hudi-client-common/src/main/java/org/apache/hudi/config/HoodieIndexConfig.java

```diff
@@ -287,6 +287,11 @@ public class HoodieIndexConfig extends HoodieConfig {
       .withDocumentation("Only applies if index type is BUCKET. Determine the number of buckets in the hudi table, "
           + "and each partition is divided to N buckets.");
 
+  public static final ConfigProperty<Boolean> BUCKET_PARTITIONER = ConfigProperty
+      .key("hoodie.bucket.remote.partitioner.enable")
+      .defaultValue(true)
```

Comment:
Default enable this new remote partitioner

Reply:
`hoodie.bucket.remote.partitioner.enable` -> `hoodie.bucket.index.remote.partitioner.enable`?